js/libs/form.js

```diff
@@ -519,6 +519,11 @@ Fliplet.Widget.instance('form-builder', function(data) {
               record = { data: record };
             }
 
+            record.data = _.omitBy(record.data, function (value) {
+              return _.isNil(value)
+                || (_.isObject(value) && _.isEmpty(value))
+                || _.isString(value) && !value.length;
```

Comment:
shouldn't the last line be all in a single parenthesis?

Reply:
It should, indeed